bigquery/tables.js

```diff
@@ -243,6 +265,26 @@ cli
   }, function (options) {
     program.exportTableToGCS(utils.pick(options, ['dataset', 'table', 'bucket', 'file', 'format', 'gzip']), utils.makeHandler());
   })
+  .command('insert <json_or_file> <dataset> <table>',
+    'Insert a JSON array (as a string or newline-delimited file) into a BigQuery table.', {},
+    function (options) {
+      var content = fs.readFileSync(options.json_or_file);
```

Comment:
If the file cannot be found, this will throw an error, so it needs to be wrapped in a try/catch, and in your catch you would just do `content = options.json_or_file;`